_shared/project/Makefile

```
@@ -44,6 +44,13 @@ shell: python
 	@pyenv exec tox -qe dev
 {% endif %}
 
+{% if cookiecutter.get("db") == "yes" %}
+.PHONY: sql
+$(call help,make sql,"Connect to the dev database with a psql shell")
+sql: python
+	@tox -qe dockercompose -- exec postgres psql --pset expanded=auto -U postgres
+{% endif %}
+
```

This is adding an unwanted blank line if you run `make template checkout=add-make-sql` in a project that doesn't have `"db"` (e.g. https://github.com/hypothesis/cookiecutter-pyramid-app-test). I think this should fix it:

```suggestion
{% if cookiecutter.get("db") == "yes" %}
.PHONY: sql
$(call help,make sql,"Connect to the dev database with a psql shell")
sql: python
	@tox -qe dockercompose -- exec postgres psql --pset expanded=auto -U postgres

{% endif %}
```

---

Took me a couple of looks there to work that out. For what ever reason Github doesn't show the trailing space in the "from" side of the diff?